docs/IPAddress-Math.rst

```diff
@@ -0,0 +1,99 @@
+IP Address Math
+===============
+
+Too frequently the existing implementation of the C# ``IPAddress`` object is too limited for anything beyond some of the most trivial interactions. Mathematical operations in fact are wholly absent, forcing developers to directly manipulate bytes [#Gulliver_001]_, often requiring a great deal of manual implementation of non-existent byte math. Don't worry though, Arcus is here to fill in some of those gaps.
```

Comment:
[Footnotes require spaces ](http://docutils.sourceforge.net/docs/user/rst/quickref.html#footnotes)

Reply:
Should we be using the auto number format?

Reply:
Used the named ones allow us to cross link across the entire set of documents. Plus it adds a 
friendly anchor.

The only reason I'm using `Gulliver_001` is because a Gulliver reference already exits. Changing it to a friendlier `GulliverLib`